oaknational/Oak-Web-Application
# LESQ-326 fix(lesson overview): additional resource the last link in the lesson…

… nav

## Description

move additional resource link in the lesson overview nav to the bottom of the list

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-1939--oak-web-application.netlify.thenational.academy/teachers/programmes/geography-secondary-ks4/units/test-ingest-this-unit/lessons/lesson-2-in-ingest-this-unit#additionalMaterial
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
